backend/.env.example

```diff
@@ -1,4 +1,5 @@
 OPEN_AI_KEY=
+OPEN_AI_SERVER_URL=https://api.openai.com/v1
```

Comment:
Let's remove the default value from here and move it to the config.go as `envDefault` value (see my other comments below).